Python/di_sensors/easy_line_follower.py

```diff
@@ -296,13 +296,13 @@ def position_val(self):
         """
         Same as calling :py:meth:`~di_sensors.easy_line_follower.EasyLineFollower.read` method like ``read("weighted-avg")``.
 
-        :rtype: float, int
```

Comment:
Leaving it as a float is not accurate. It should be like this:
`float/int`
or
`float (for 0->1) or int (for 2 or 3)`

And I wouldn't convert the 2s and 3s to float, because you could get into some precision differences. So leave them be like this and adjust the return type.